lib/containers/k8s.pm

```diff
@@ -56,6 +62,7 @@ sub install_k3s {
     script_run("mkdir -p ~/.kube");
     script_run("rm -f ~/.kube/config");
     assert_script_run("ln -s /etc/rancher/k3s/k3s.yaml ~/.kube/config");
+    record_info('cmd', "k3s installed");
```

Comment:
```suggestion
    record_info('k3s', "k3s installed");
```

Let's give the user some more context in the info box!

Reply:
done